src/routes/backupTarget/BackupTargetList.js

```diff
@@ -0,0 +1,113 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { Table, Icon, Tooltip } from 'antd'
+import BackupTargetActions from './BackupTargetActions'
+import { pagination } from '../../utils/page'
+
+function list({ loading, dataSource, deleteBackupTarget, editBackupTarget, rowSelection, height }) {
+  const columns = [
+    {
+      title: 'Name',
+      dataIndex: 'name',
+      key: 'name',
+      width: 150,
+      sorter: (a, b) => a.name.localeCompare(b.name),
+      render: (text) => {
+        return (
+          <div>{text}</div>
+        )
+      },
+    }, {
+      title: 'URL',
+      dataIndex: 'backupTargetURL',
+      key: 'backupTargetURL',
+      width: 250,
+      sorter: (a, b) => a.backupTargetURL.localeCompare(b.backupTargetURL),
+      render: (text) => {
+        return (
+          <div>{text}</div>
+        )
+      },
+    }, {
+      title: 'Credential Secret',
+      dataIndex: 'credentialSecret',
+      key: 'credentialSecret',
+      width: 150,
+      sorter: (a, b) => a.credentialSecret.localeCompare(b.credentialSecret),
+      render: (text) => {
+        return (
+          <div>{text}</div>
+        )
+      },
+    }, {
+      title: 'Poll Interval',
+      dataIndex: 'pollInterval',
+      key: 'pollInterval',
+      width: 100,
+      sorter: (a, b) => a.pollInterval.localeCompare(b.pollInterval),
+      render: (text) => {
+        return (
+          <div>{text}</div>
+        )
+      },
+    }, {
+      title: 'Available',
```

Comment:
[Discussion]
Instead of using a boolean `Available: True/False`, would it be clearer to display `Status: Available/Error`, following the pattern in other pages?

Reply:
For the query string, `booleanValue=true` seems too generic.
Should we treat an empty `booleanValue=` as true or false? Also, should `booleanValue=True` be considered valid?
Switching to `status` would eliminate these concerns.

Reply:
Then we should update available column ot status column and filter status available / error.

Reply:
Yeah, I don't have strong opinions on the status value, depends on if there will be other statuses in the future. 
What do you think?

Reply:
Modifiied to status column with available / error text.